pkgs/applications/misc/cura/lulzbot.nix

```diff
@@ -1,58 +1,66 @@
-{ stdenv, fetchurl, dpkg, bash, python27Packages }:
+{ lib, fetchgit, curaengineLulzbot, cmake, jq, python3Packages, qtbase, qtquickcontrols2 }:
 
 let
-  py = python27Packages;
+  # admittedly, we're using (printer firmware) blobs when we could compile them ourselves.
+  curaBinaryDataVersion = "3.6.18"; # Marlin v2.0.0.144. Keep this accurate wrt. the below.
+  curaBinaryData = fetchgit {
+    url = https://code.alephobjects.com/diffusion/CBD/cura-binary-data.git;
+    rev = "cdc046494bbfe1f65bfb34659a257eef9a0100a0";
+    sha256 = "0v0s036gxdjiglas2yzw95alv60sw3pq5k1zrrhmw9mxr4irrblb";
+  };
+  curaengine = curaengineLulzbot;
+  libarcus = python3Packages.libarcusLulzbot;
+  uranium = python3Packages.uraniumLulzbot;
+  libsavitar = python3Packages.libsavitarLulzbot;
```

Comment:
Since we don't want to use these python packages in any other place, it might be better to copy them to `pkgs/applications/misc/cura/lulzbot` and just reference them here locally.
The reason is that having different versions of a python library will collide in `PYTHONPATH`, which is why we want to avoid having different versions of the same library for python.

Reply:
That's reasonable. If the Ultimaker branches of these libraries had no other users (something I haven't yet looked to confirm/deny), would we want to do the same for them?

Reply:
@Mic92, is 784c69adfad9d5b1c1e29bbaf34e1a6127268f66 in line with your suggestion here?